libs/scully-schematics/src/scully/index.ts

```diff
@@ -47,7 +47,7 @@ const createScullyConfig = (options: Schema) => (tree: Tree, context: SchematicC
       `import { ScullyConfig } from '@scullyio/scully';
 
 /** this loads the default render plugin, remove when switching to something else. */
-import '@scullyio/scully-plugin-puppeteer';
+import '${options.renderer === 'puppeteer' ? '@scullyio/scully-plugin-puppeteer' : '@scullyio/scully-plugin-playwright'}';
```

Comment:
add something that skips this when sps is selected plase.

Reply:
I've updated this